.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,101 @@
+---
+name: "Bug Report"
+about: Report a bug
+labels: bug, needs-triage
+
+---
+<!--
+Thanks for filing an issue!  Please provide as much of the information requested below as possible.
+
+To expedite issue processing please search open and closed issues before submitting a new one.
+Existing issues often contain information about workarounds, resolution, or progress updates.
+
+Even better, we'd love a pull request to resolve the issue!
+-->
+
+# Bug Report
+
+### Description
+
+<!-- A clear and concise description of the problem. -->
+
+### Schema
+
+<!--
+Is this issue related to a specific shema?  Has the schema already been contirbuted? Can the schema be contribted for reproduction?  
+
+Provide the content of the schema or a link to the contributed schema if possible.
+-->
+
+### Is this a regression?
+
+<!-- Did this behavior work in a previous version?  What version? -->
+
+### Steps to reproduce
+
+<!-- Detailed steps for how to reproduce the observed behavior. -->
+
+### Exception or error
+
+<!-- Is there a specific exception or error message being seen? -->
+
+### Environment
```

Comment:
@grighetto @absurdfarce can you all let me know if there's any other environmental items you'd like captured here?  I did my best to guess at what would be useful, but you all will know far more than I here, so just let me know.